nix/m1-support/u-boot/default.nix

```diff
@@ -21,20 +21,29 @@ in (buildPkgs.buildUBoot rec {
   extraMeta.platforms = [ "aarch64-linux" ];
   filesToInstall = [
     "u-boot-nodtb.bin.gz"
-    "m1n1-u-boot.macho"
     "m1n1-u-boot.bin"
   ];
   extraConfig = ''
     CONFIG_IDENT_STRING=" ${version}"
+    CONFIG_VIDEO_FONT_4X6=n
+    CONFIG_VIDEO_FONT_8X16=n
+    CONFIG_VIDEO_FONT_SUN12X22=n
+    CONFIG_VIDEO_FONT_TER12X24=n
+    CONFIG_VIDEO_FONT_TER16X32=y
   '';
 }).overrideAttrs (o: {
   # nixos's downstream patches are not applicable
-  patches = [ ];
+  # however, we add in bigger u-boot fonts because the mac laptop screens are high-res
+  # these patches obtained via:
+  # https://git.alpinelinux.org/aports/tree/testing/u-boot-asahi
+  patches = [ 
+    ./apritzel-first5-video.patch
+    ./mps-u-boot-ter12x24.patch 
```

Comment:
I would prefer to use `fetchpatch` to download these (ideally from the original source) instead of vendoring them as they are pretty large.

Reply:
I originally wrote this using `fetchpatch`, but I was worried about the alpine testing branch I obtained this code from disappearing suddenly.

Reply:
AFAICT this is not a separate branch but a testing folder. If you pin to a specific commit using the [plain link](https://git.alpinelinux.org/aports/plain/testing/u-boot-asahi/mps-u-boot-ter12x24.patch?id=a211444892fc1fea63151af46d544fda345fa492) I can't see why it should disappear. There are a few places in nixpkgs where alpinelinux's aports repo is `fetchpatch`ed from directly too.